### ISSUE TITLE
Title history tweaks

### ISSUE THREAD

**Issue by loup99** [loup99/BP#64]
- [ ] Split the Arochi tribe to represent the Aethelrugi and Ranii as well (see #54 for area)
- [ ] Add the Salihids at start, who preceded the Ghassanids

**Comment by loup99** [comment]
Adding a note on Septem, the presence of an independent Roman realm in 476 should be reviewed, what source is it based upon?

**Comment by Abian36** [comment]
Precisely, it is based on a lack of sources. I remember discussing its history back when I was tweaking the implementation of Spania in CK2. It is uncertain when the VIsigoths crossed and conquered or if the Vandals really held it directly or if it was more of like a mostly independent entity, nominally vassal to them. I assume the Roman duke was a placeholder that was simply left there for convenience.
